src/Locations/PageParentTemplate.php

```diff
@@ -23,10 +23,12 @@ function initialize() {
    * @inheritdoc
    */
   function rule_match($result, $rule, $screen) {
-    $post_parent_id = wp_get_post_parent_id($screen['post_id']);
-    $post_parent_template = get_page_template_slug($post_parent_id);
-    $match = $this->compare($post_parent_template, $rule);
-    return $match;
+    if (isset($screen['post_id'])) {
+      $post_parent_id = wp_get_post_parent_id($screen['post_id']);
+      $post_parent_template = get_page_template_slug($post_parent_id);
+      $match = $this->compare($post_parent_template, $rule);
+      return $match;
+    }
```

Comment:
Not sure what I did there when I created it 😅, but looks like this is a filter and thus a value must be returned. I think it could be rewritten like this:

```suggestion
    if (isset($screen['post_id'])) {
      $post_parent_id = wp_get_post_parent_id($screen['post_id']);
      $post_parent_template = get_page_template_slug($post_parent_id);
      $result = $this->compare($post_parent_template, $rule);
    }
	return $result;
```